packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -84,55 +98,62 @@ class MLKitScannerPageState extends State<MLKitScannerPage> {
       scale = 1 / scale;
     }
 
-    return Scaffold(
-      body: ScannerOverlay(
-        restartCamera: _resumeImageStream,
-        stopCamera: _stopImageStream,
-        model: _model,
-        scannerWidget: Transform.scale(
-          scale: scale,
-          child: Center(
-            child: CameraPreview(
-              _controller!,
-            ),
-          ),
+    return Transform.scale(
+      scale: scale,
+      child: Center(
+        key: ValueKey<bool>(stoppingCamera),
+        child: CameraPreview(
+          _controller!,
         ),
       ),
     );
   }
 
   Future<void> _startLiveFeed() async {
+    stoppingCamera = false;
     final CameraDescription camera = cameras[_cameraIndex];
-    _controller = CameraController(
+
+    final CameraController cameraController = CameraController(
       camera,
       ResolutionPreset.high,
```

Comment:
have we tested other resolutions ? does it influence the scanning a lot ? it could alleviate some of the resource concerns

Reply:
No not tested in any way, just took high that we have a "high" resolution, we fill nearly the full screen with the preview but not max so that we don't spend too much performance. After all it's just the preview not how much we get from the camera which very likely needs more performance.